BlazarBase/src/main/java/com/hubspot/blazar/base/Module.java

```diff
@@ -97,6 +98,17 @@ public Module withId(int id) {
   }
 
   @Override
+  public String toString() {
+    return MoreObjects.toStringHelper(this)
+        .add("id", id)
+        .add("name", name)
+        .add("type", type)
+        .add("path", path)
+        .add("glob", glob)
+        .add("buildpack", buildpack).toString();
```

Comment:
Talked offline, answer is: the buildpack object here is a reference to the branch that the buildpack resides on.
A module can have an associated buildpack